file/file.h

```diff
@@ -53,8 +53,12 @@ SOFTWARE.
 namespace bricks {
 
 // Platform-indepenent, injection-friendly filesystem wrapper.
-// TODO(dkorolev): Move the above methods under FileSystem.
 struct FileSystem {
+#ifndef BRICKS_WINDOWS
+  static constexpr char PathSeparatingSlash = '/';
+#else
+  static constexpr char PathSeparatingSlash = '\\';
+#endif
   static inline std::string GetFileExtension(const std::string& file_name) {
     const size_t i = file_name.find_last_of("/\\.");
```

Comment:
This string should likely use the platform-dependent `PathSeparatingSlash` instead of both. Don't forget to add the dot to it.